app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -134,31 +134,31 @@ def parse_node(node)
       new_result.merge!(
         :type                       => 'ManageIQ::Providers::Kubernetes::ContainerManager::ContainerNode',
         :identity_infra             => node.spec.externalID,
-        :identity_machine           => node.status.nodeInfo.machineID,
-        :identity_system            => node.status.nodeInfo.systemUUID,
-        :container_runtime_version  => node.status.nodeInfo.containerRuntimeVersion,
-        :kubernetes_proxy_version   => node.status.nodeInfo.kubeProxyVersion,
-        :kubernetes_kubelet_version => node.status.nodeInfo.kubeletVersion,
+        :identity_machine           => node.status.try(:nodeInfo).try(:machineID),
```

Comment:
@zakiva you can try to extract this into something like:

    nodeInfo = node.status.try(:nodeInfo)

    if nodeInfo
      new_result.merge!(
        :identity_machine           => node.status.nodeInfo.machineID,
        ...
      )
    end

So then down below you can reuse `nodeInfo` also for other purposes:

    :distribution   => nodeInfo.try(:osImage),
    ...